components/ToastMessage/components/ActionToast.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { connectStyle } from '@shoutem/theme';
+import { images } from '../assets';
+import BaseToast from './BaseToast';
+
+function ActionToast({ style, isVisible, props }) {
+  return (
+    <BaseToast
+      customToastStyle={style}
+      imageSource={images.action}
+      isVisible={isVisible}
+      {...props}
```

Comment:
```suggestion
function ActionToast({ style, isVisible, ...otherProps }) {
  return (
    <BaseToast
      customToastStyle={style}
      imageSource={images.action}
      isVisible={isVisible}
      {...otherProps}
```

Just a suggestion, I've seen this notation elsewhere and it makes more sense to me because `style` and `isVisible` are also props, so having them next to a prop called `props` seems unusual.